### ISSUE TITLE
Is Vugu ready for side-project/MVP application usage?

### ISSUE THREAD

**Issue by forstmeier** [vugu/vugu#115]
**Question**
I've been tinkering on a database app side project and I've built the backend entirely in Go. That backend will be exposing a GraphQL API and I was looking to build a smallish UI wrapper around it. There would only be a few key pages (e.g. singup/login, user profile, org admin, and database query).

My question: is Vugu currently strong enough to support an app like this with the necessary routing and some page access security? I'm not that worried about binary sizes/speed at the moment. I'd love to use something like this (my alternative would be Vue) for my project.

**Is your question related to a problem? Please describe.**
TL;DR: _can Vugu work with a small, couple-page MVP app on top of a GraphQL backend API?_

**Suggested Changes**
N/A

**Additional context**
No additional context, but I love the idea of this project!

**Comment by bradleypeabody** [comment]
Sounds like an interesting project.

> is Vugu currently strong enough to support an app like this with the necessary routing and some page access security?

It should have the basic pieces, yes; including a router and you can control what shows/hides based on routes or other information from the backend.

That said, I can say that it is likely you will run into unforeseen issues.  If you do proceed with this, I'll do my best to help fix what comes up, but obviously there's risk involved with such an undertaking.  I think the decision to use Vugu for your project will have to be based at least partly on if you'll have the time and patience to work through the issues that could come up.

That's my warning.  But otherwise I think it would be great!  If you decide to proceed, feel free to reach out with further questions.   I'm also very close to putting up a big documentation update and releasing v0.2.0 - but for the moment for any new code please see https://github.com/vugu/vugu/wiki/Refactor-Notes---Sep-2019

**Comment by itstmyi** [comment]
I built several small projects successfully based on vugu@v0.0.0-20200322002911-7d3a6c0162c2 but failed based on the latest version .
